components/PostContent/PostContent.tsx

```diff
@@ -60,6 +61,18 @@ export default function PostContent( props: {
 								/>
 							);
 
+						case 'core/quote':
+							const quoteProps = {
+								...defaultProps,
+								large: blockProps.className || undefined,
```

Comment:
If we're mapping to a prop named `large`, we should do the equality test here and make the prop boolean. Otherwise, we should name the prop `className` and keep the logic in the component. (The latter seems preferable to keep `PostContent` as pure as possible.)